src/conversions.jl

```diff
@@ -248,6 +249,21 @@ function convert_arguments(P::PointBased, x::Rect3D)
     convert_arguments(P, decompose(Point3f0, x)[inds])
 end
 
+edges(v::AbstractVector) = edges(range(extrema(v)...; length=length(v)))
+
+function edges(centers::AbstractRange)
+    min, s, l = minimum(centers), step(centers), length(centers)
+    return range(min - s / 2, step=s, length=l + 1)
+end
+
+function adjust_axes(::DiscreteSurface, x::AbstractVector{<:Number}, y::AbstractVector{<:Number}, z::AbstractMatrix)
```

Comment:
This is the only scenario where the conversion behavior differs between discrete and continuous surfaces, not sure if there are others that I'm missing.